controllers/authController.js

```diff
@@ -13,7 +14,8 @@ exports.login = async (req, res) => {
       })
     }
 
-    if (req.body.password !== user.password) {
+    req.body.password = bcrypt.hashSync(req.body.password, constants.saltRounds)
```

Comment:
No need of this

Reply:
Daca scot linia aia nu va lasa in baza de date parola in plaintext?

Creerea userului se face cu  `const user = await req.db.User.create(req.body)`

Reply:
E la login asta